src/util/config.cpp

```diff
@@ -201,21 +199,25 @@ static std::vector<std::string> find_disallowed_keys(const JAST& json,
   return disallowed;
 }
 
-bool init(const std::string& wakeroot_path) {
+bool WakeConfig::init(const std::string& wakeroot_path) {
   if (_config != nullptr) {
     std::cerr << "Cannot initialize config twice" << std::endl;
-    assert(false);
+    exit(1);
```

Comment:
nit: I prefer assert here since this is a "code failure" not a "user failure". It'll point to the exact line instead of needed to grep for "Cannot initialize config twice". And if we ever see this error then its our fail so it should blow up

Reply:
It doesn't always fail though because a lot of times asserts don't trigger.

Reply:
strange. I use this pattern frequently. This probably deserves a grep across the repo then. (Also what good are asserts if we can't  trust them?)

Reply:
They do trigger if you set the DEBUG define is set correctly but you have to make sure its set correctly in every translation unit (and it wasn't set in this one)